internal/controller/operatorconfigmap_controller.go

```diff
@@ -950,3 +987,59 @@ func (c *OperatorConfigMapReconciler) deleteDelegatedCSI() error {
 	}
 	return nil
 }
+
+func (c *OperatorConfigMapReconciler) getCsiCustomTolerations() ([]corev1.Toleration, []corev1.Toleration, error) {
+	nodePluginTolerations, controllerPluginTolerations := []corev1.Toleration{}, []corev1.Toleration{}
+
+	// list the rook ceph cm
+	rookCephOperatorConfig := &corev1.ConfigMap{}
+	err := c.Client.Get(c.ctx, types.NamespacedName{Name: utils.RookCephOperatorConfigName, Namespace: c.OperatorNamespace}, rookCephOperatorConfig)
+	if err != nil {
```

Comment:
Please use the local get util
```suggestion
	rookCephOperatorConfig := &corev1.ConfigMap{}
	rookCephOperatorConfig.Name = utils.RookCephOperatorConfigName
	rookCephOperatorConfig.Namespace = c.OperatorNamespace
	if err := c.get(rookCephOperatorConfig); err != nil {
```
```